madmom/features/beats_hmm.py

```diff
@@ -462,6 +495,10 @@ class RNNBeatTrackingObservationModel(ObservationModel):
     observation_lambda : int
         Split one beat period into `observation_lambda` parts, the first
         representing beat states and the remaining non-beat states.
+    obslik_floor : float
+        Observation likelihood floor. Values below this threshold and
+        above 1-threshold are clipped.
+
```

Comment:
I think ``clip`` is a better name, since it does exactly this. We could also think of allowing a tuple with individual lower and upper clipping values.

Again, please do not insert additional empty lines.